src/components/Pages/OfficialDocuments/OfficialDocumentEntry.js

```diff
@@ -2,10 +2,11 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { injectIntl } from 'react-intl';
 import moment from 'moment-timezone';
+import { Link } from 'react-router-dom';
 
 import { officialdocuments as i18n } from 'js/i18n/definitions';
 
-const OfficialDocument = ({ page: {id, date, title, authoringOffice, summary, name, link, pdfSize }, intl }) => {
+const OfficialDocumentEntry = ({ page: {id, date, title, authoringOffice, summary, name, link, pdfSize, slug }, intl }) => {
```

Comment:
Another renaming. This is the entry in the list of official documents. Also open to name suggestions if Entry doesnt work. It's also what I named the list item in the Event Lists, so that's why I went with entry.